abstract/managers/TelemetryManager.js

```
@@ -0,0 +1,177 @@
+      return;
+    }
+
+    this._config[key] = value;
+  }
+
+  /**
+   * - @param {Pick<TelemetryState, 'eventType' | 'payload'| 'config'> } body
+   * - @returns {TelemetryState}
+   */
+  _formatingPayload(body) {
+    if (body.payload?.activity) delete body['payload']['activity'];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)